rust/xaynet-analytics/src/lib.rs

```diff
@@ -8,7 +8,11 @@
 //! a framework that allows mobile applications to collect and aggregate
 //! analytics data via the _Privacy-Enhancing Technology_ (PET) protocol.
 
+#[cfg(not(tarpaulin))]
```

Comment:
Do we need this since this crate is commented in the manifest?
Same question for the `--exclude ` option in the ci, why it is needed? tarpaulin does not look at the `members` variable but check all directories anyway?

Reply:
ok thanks. Can you maybe open an issue on their repo? I think it is a bug.

Reply:
sure, we should have a minimum runnable example for them to showcase this, i'll put it on my todo list.